webapp/packages/plugin-datasource-transaction-manager/src/actions/ACTION_COMMIT.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createAction } from '@cloudbeaver/core-view';
+
+export const ACTION_COMMIT = createAction('commit', {
+  label: 'plugin_datasource_transaction_manager_commit',
+  tooltip: 'plugin_datasource_transaction_manager_commit',
+  icon: '/icons/commit_m.svg',
+});
```

Comment:
If an action is specific to some application part, please don't use common names: ACTION_SAVE, ACTION_DELETE, etc.

please rename: `export const ACTION_DATASOURCE_TRANSACTION_COMMIT = createAction('datasource-transaction-commit'`